pep/app/services/ajax.ts

```diff
@@ -56,7 +56,6 @@ export default class AjaxService extends Service {
      */
     async request<T>(url: string, options: RequestInit = {}): Promise<T> {
         setProperties(options, {
-            credentials: 'include', //NOTE: only needed if cookies must be sent in API requests (which are needed for auth right now)
```

Comment:
just noting that i think we will ultimately need this left in (as alan mentioned the apis will want to be sending/sharing cookies for some other stuff), so if if its causing issue with PaDs, we may need to have him address that. For the time being, i'd probably just comment this out instead of deleting it.